docs/FAQ.md

```diff
@@ -0,0 +1,94 @@
+# Frequently asked questions
+
+### Why does my branch not move to the new commit after `jj new/commit`?
+
+If you're familiar with Git, you might expect the current branch to move forward
+when you commit. However Jujutsu does not have a concept of a "current branch".
+
+To move branches, use `jj branch set`. 
+
+### I made a commit and `jj git push --all` says "Nothing changed" instead of pushing it. What do I do?
+
+`jj git push --all` pushes all _branches_, not all revisions. You have two 
+options: 
+
+* Using `jj git push --change` will automatically create a branch and push it.
+* Using `jj branch` commands to create or move a branch to either the commit 
+you want to push or a descendant on it. Unlike Git, Jujutsu doesn't do this 
+automatically (see previous question).
+
+### Where is my commit, why is it not visible in `jj log`? 
+
+Is your commit visible with `jj log -r 'all()'`? 
+
+If yes, you should be aware that `jj log` only shows the revisions matching 
+`revsets.log` by default. You can change it as described in [config] to show 
+more revisions. 
+
+If not, the revision may have been abandoned (e.g because you used `jj abandon`
+, or because it's an obsolete version that's been rewritten with `jj rebase`, 
+`jj describe`, etc) In that case, `jj log -r commit_id` should show the 
+revision as "hidden". `jj new commit_id` or `jj edit commit_id` should make the
+revision visible again.
+
+See [revsets] and [templates] for further guidance.
+
+### How do I add a file? 
```

Comment:
I think I figured out what I wanted to say here. The question could say:

> ### Can I prevent jj from recording my unfinished work? I'm not ready to commit it.

The answer could be:

> Jujutsu automatically records new files in the current working-copy commit and doesn't provide a way to prevent that.

> However, you can easily record intermediate drafts of your work. If you think you might want to go back to the current state of the working-copy commit, simply use `jj new`. There's no need for the commit to be "finished" or even have a description.
>
> Then, future edits will go into a new working-copy commit on top of the now former working-copy commit. Whenever your are happy with another set of edits, use `jj squash` to amend the previous commit.

Reply:
done